common/lib/api-client/auth.js

```diff
@@ -49,9 +50,14 @@ Auth.prototype = {
       },
     }
 
+    debug('AUTH REQUEST', this.config.authUrl)
     return axios
       .post(this.config.authUrl, data, config)
       .then(response => response.data)
+      .catch(error => {
+        debug('AUTH ERROR', error)
```

Comment:
Is there much benefit in us adding this extra debugging? In the case of an error, the express error handler would show that anyway, no?

Reply:
Not in the case of anything that we don't rethrow to the express error handler - eg. the 404s and 422s for the person images.

I think this is a massive improvement since by default we do not drown in devour messages - only if we explicitly ask for them. (Whereas before the choice was to pretend it wasn't DEV or hack devour to exclude certain cases). I have already found this much more useful and agreeable

(NB. the catch block will also be used for the instrumentation to record the failure)